README.md

```diff
@@ -1,2 +1,28 @@
 # jest-evaluator-action
-Jest evaluator Action for Tryber projects
+Jest evaluator action for Tryber projects
+
+This action evaluate Tryber projects with [Jest](https://jestjs.io/) library.
+
+## Inputs
```

Comment:
Why are the inputs and outputs topics empty?

Reply:
Input is empty because there is no input. Output is not empty.

Reply:
So the inputs shouldn't be removed from the readme? Or at least contain a text saying that there is no inputs for this action?

Reply:
I think this is like a template do describe how actions works.